recipes/api.rb

```diff
@@ -6,10 +6,11 @@
 include_recipe 'apiaxle::_setup'
 include_recipe 'apiaxle::_config'
 
-%w(repl api).each do |p|
-  nodejs_npm "apiaxle-#{p}" do
-    version node[:apiaxle][:version]
-  end
+nodejs_npm 'apiaxle-repl' do
+  version node[:apiaxle][:version]
+end
+nodejs_npm 'apiaxle-api' do
```

Comment:
should we do a conditional here so this recipe still works even if you don't want the forked version